src/interactive/Dropdown.jsx

```diff
@@ -230,7 +243,8 @@ Dropdown.propTypes = {
 	isActive: PropTypes.bool,
 	manualToggle: PropTypes.func,
 	maxWidth: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-	minWidth: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+	minWidth: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+	shouldPortal: PropTypes.bool,
```

Comment:
i know this is already merged, but `noPortal` seems like it would be a better prop name because that's what the CSS classname variant uses, and since by default it _will_ portal, it allows callers to opt-in to 'no portalling' with `<Dropdown noPortal />`